job-descriptions/engineering-manager-web.md

```diff
@@ -2,17 +2,20 @@
 
 # Engineering Manager - Web
 
-We are looking for an experienced engineering leader for our [web team](https://about.sourcegraph.com/handbook/engineering/web).
+We are looking for an experienced engineering leader for our [web team](https://about.sourcegraph.com/handbook/engineering/web). This person will facilatate all web development for Sourcegrap and be responsible driving the team towards intended growth mentioned in the handbook (grow the team until it's large enough to split into two teams, web infrastructure, and extensions and integrations).
 
 ## Qualifications
 
-- 2 years of experience managing a team of software engineers.
-- 2 years of experience coding web applications and/or browser extensions.
+- 2+ years of experience directly managing at least 3 full-time software engineers.
+- Experience goal setting, prioritizating work for team, and conducting 1:1s. 
+- Experience developing and managing development teams for web applications and/or browser extensions.
```

Comment:
> Experience developing and managing development teams

I assume the intent here is to say we are looking for someone with both IC and manger experience? If so, I think the language is a bit ambiguous and it would be clearer to separate. What do you think?

```suggestion
- Experience building web applications and/or browser extensions.
- Experience managing teams building web applications and/or browser extensions.
```